tests/unit/api/port_revert.py

```diff
@@ -43,13 +43,6 @@ def tearDown(self):
         self.request_context.pop()
         releaseDB()
 
-    def test_no_nic(self):
```

Comment:
Might be nice to put something into place that replaces this, i.e. verify that port_revert *does* work when there's no nic.